numpy/core/setup.py

```diff
@@ -394,7 +391,11 @@ def visibility_define(config):
 
 def configuration(parent_package='',top_path=None):
     from numpy.distutils.misc_util import Configuration, dot_join
-    from numpy.distutils.system_info import get_info
+    from numpy.distutils.system_info import get_info, blas_opt_info
+
+    # Accelerate is buggy and not supported
+    if 'accelerate' in blas_opt_info.blas_order:
+        blas_opt_info.blas_order.remove('accelerate')
```

Comment:
I'm tempted to remove it from lapack here too, to keep it in one place

Reply:
makes sense

Reply:
You mean move the corresponding change to `lapack_opt_info` out of `numpy/linalg.setup.py` to here? That seems fragile.

Reply:
I repeated the code in both `numpy/core/setup.py` and `numpy/linalg/setup.py`, with comments.